### PR TITLE
Navbar and Footer Responsiveness

### PR DESCRIPTION
**Pull request by JDVariadic**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -23,17 +23,23 @@
   <nav>
     <div class="text-container">
       <div class="navigation-primary">
-        <img src="../assets/images/image 1.png" />
+        <img src="assets/images/image 1.png" />
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 2]
```diff
@@ -23,17 +23,23 @@
...
         <span>FINISHERS</span>
-        <img src="../assets/images/image 2.png" />
+        <img src="assets/images/image 2.png" />
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 3]
```diff
@@ -87,7 +93,7 @@ <h2>
         </div>
         <div class="people-collection-child">
           <div>
-            <img src="../assets/images/Ellipse 2.png" />
+            <img src="assets/images/Ellipse 2.png" />
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 4]
```diff
@@ -77,7 +83,7 @@ <h2>
         <!--INSERT PEOPLE HERE-->
         <div class="people-collection-child">
           <div>
-            <img src="../assets/images/Ellipse 2.png" />
+            <img src="assets/images/Ellipse 2.png" />
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 5]
```diff
@@ -97,7 +103,7 @@ <h2>
         </div>
         <div class="people-collection-child">
           <div>
-            <img src="../assets/images/Ellipse 2.png" />
+            <img src="assets/images/Ellipse 2.png" />
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 6]
```diff
@@ -107,7 +113,7 @@ <h2>
...
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 7]
```diff
@@ -118,7 +124,7 @@ <h2>
 
         <div class="people-collection-child">
           <div>
-            <img src="../assets/images/Ellipse 2.png" />
+            <img src="assets/images/Ellipse 2.png" />
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 8]
```diff
@@ -129,7 +135,7 @@ <h2>
...
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 9]
```diff
@@ -139,7 +145,7 @@ <h2>
         </div>
         <div class="people-collection-child">
           <div>
-            <img src="../assets/images/Ellipse 2.png" />
+            <img src="assets/images/Ellipse 2.png" />
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 10]
```diff
@@ -149,7 +155,7 @@ <h2>
...
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 11]
```diff
@@ -159,7 +165,7 @@ <h2>
...
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 12]
```diff
@@ -169,7 +175,7 @@ <h2>
...
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 13]
```diff
@@ -179,7 +185,7 @@ <h2>
...
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 14]
```diff
@@ -189,7 +195,7 @@ <h2>
...
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/index.html [hunk 15]
```diff
@@ -209,7 +215,7 @@ <h2>
   <footer>
     <div class="footer-container">
       <div>
-        <img src="../assets/images/image 1.png" />
+        <img src="assets/images/image 1.png" />
```
Comment (Hjkun77) [inline public/index.html]: add `alt="..."`

File: public/scripts/index.js [hunk 16]
```diff
@@ -0,0 +1,12 @@
+const navToggle = () => {
+  const secondaryNav = document.querySelector('.navigation-secondary')
```
Comment (Hjkun77) [inline public/scripts/index.js]: set this as global variable

File: public/scripts/index.js [hunk 17]
```diff
@@ -0,0 +1,12 @@
...
+  const burger = document.querySelector('.burger')
```
Comment (Hjkun77) [inline public/scripts/index.js]: set this as global variable

File: public/scripts/index.js [hunk 18]
```diff
@@ -0,0 +1,12 @@
...
+
+  burger.addEventListener('click', () => {
```
Comment (Hjkun77) [inline public/scripts/index.js]: remove this and add it as an onclick method in the html than an eventlistener

File: public/scripts/index.js [hunk 19]
```diff
@@ -0,0 +1,12 @@
...
+    console.log(secondaryNav)
+    secondaryNav.classList.toggle('nav-display')
+    console.log('working')
+  })
+}
+
+navToggle()
```
Comment (Hjkun77) [inline public/scripts/index.js]: call this function as an onclick in the html instead

File: public/scripts/index.js [hunk 20]
```diff
@@ -0,0 +1,12 @@
+const navToggle = () => {
+  const secondaryNav = document.querySelector('.navigation-secondary')
+  const burger = document.querySelector('.burger')
+
+  burger.addEventListener('click', () => {
+    console.log(secondaryNav)
```
Comment (Hjkun77) [inline public/scripts/index.js]: remove console.log()

File: public/scripts/index.js [hunk 21]
```diff
@@ -0,0 +1,12 @@
...
+    secondaryNav.classList.toggle('nav-display')
+    console.log('working')
```
Comment (Hjkun77) [inline public/scripts/index.js]: remove console.log()

**Review by Hjkun77 — CHANGES_REQUESTED** [review]
Please fix these changes first before merging

**Comment by Znarfois** [comment]
Updated the index.html to have onclick instead of inside index.js @Hjkun77

File: public/scripts/index.js [hunk 22]
```diff
@@ -0,0 +1,7 @@
+const secondaryNav = document.querySelector('.navigation-secondary');
+
+const burger = document.querySelector('.burger')
+
+const navToggle = () => {
```
Comment (Hjkun77) [inline public/scripts/index.js]: `const navToggle = () => secondaryNav.classList.toggle('nav-display')`

**Review by Hjkun77 — CHANGES_REQUESTED** [review]
since simple lang naman yung function...let's make it like this na lang hehe

**Review by Znarfois — APPROVED** [review]

**Pull request merged**